skimage/metrics/_structural_similarity.py

```diff
@@ -76,8 +79,20 @@ def structural_similarity(im1, im2,
 
     Notes
     -----
+    If the `data_range` is not specified, the range is automatically guessed
+    based on the image data type. However for floating-point image data, this
+    estimate yields a result double the value of the desired range, as the
+    `dtype_range` in `skimage.util.dtype.py` has defined intervals from -1 to
+    +1. This yields an estimate of 2, instead of 1, which is most often
```

Comment:
... for signed integers. Then the question is that of type inference.

Maybe link to https://scikit-image.org/docs/stable/user_guide/data_types.html as well.